TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Section.js

```diff
@@ -44,7 +47,7 @@ define(
 			if (!this.get('inspector.configuration.' + this.get('_nodeType'))) {
 				this.set('inspector.configuration.' + this.get('_nodeType'), {});
 			}
-			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('group'), this.get('_collapsed'));
+			this.set('inspector.configuration.' + this.get('_nodeType') + '.' + this.get('groupName'), this.get('_collapsed'));
```

Comment:
@radmiraal unfortunately, the property containing the group name in the "group" object is currently called "group" and not "groupName". Changing this would require more work on the inspector in general.